### PR TITLE
[WIP] Android updates

### PR DESCRIPTION
**Pull request by arminn**

### CODE REVIEW

File: build/device.mk [hunk 1]
```diff
@@ -74,18 +74,6 @@ PRODUCT_PROPERTY_OVERRIDES += \
...
     ro.carrier=unknown
 
-# Define audio tracks that we want to use as default.
-# Do not change method of assignment because new values have to be prepended
-# and not appended (do not use +=). Explanation from google's groups:
-# "For PRODUCT_PROPERTY_OVERRIDES ... the first instance takes precedence."
-# For same reason this override has to be used after include of car.mk
-# in order to use our values.
-PRODUCT_PROPERTY_OVERRIDES := \
-     ro.config.ringtone=Atria.ogg \
```
Comment (rshym) [inline build/device.mk:84]: In car.mk we have `ringtone=Girtab.ogg`.
And we are using AudioPackage13/mk, which doesn't copy Girtab.ogg on device.
This is the reason to use our own `PRODUCT_PROPERTY_OVERRIDES` to specify files provided by AudioPackage13.

**Comment by rshym** [comment]
Misprint in word `chnges` in last commit `Comment input mapping`.
Maybe commit message can be reworded, because it's not clear for now what commit does.
Maybe, just maybe, something like "Do not use input mapping because it is different for different versions of kernel".
Sorry if my proposition is wrong.

File: build/device.mk (hunk 1 above)
Comment (arminn) [reply build/device.mk:84]: Ok, so then we need to try PRODUCT_PROPERTY_OVERRIDES += or rethink how to solve it. In the future version, the build system will  throw error on double := to PRODUCT_PROPERTY_OVERRIDES.

Comment (rshym) [reply build/device.mk:84]: Unfortunately, we can use append (+=) due to Google's limitation "For PRODUCT_PROPERTY_OVERRIDES ... the first instance takes precedence". So if we want our change to work - we need to put our setting before same change in car.mk (as it is in discussed code).
But if this change is not allowed in next version of build system we need to change code in car.mk

**Pull request closed without merging**